appveyor.yml

```diff
@@ -3,15 +3,4 @@ version: '{build}'
 install:
     - configure
 
-build: off
-
-
-test_script:
```

Comment:
You cannot remove the tests run... this would be a bit like hurting yourself with a knife intentionally.  
Instead only remove the on_success and on_failure block below

Reply:
Thank you, I got it; it was my fault.